source/_integrations/braviatv.markdown

```diff
@@ -39,28 +39,7 @@ The integration supports `remote` platform. The remote allows you to send key co
 The commands that can be sent to the TV depends on the model of your TV. To display a list of supported commands for your TV, call the service `remote.send_command` with non-valid command (e.g. `Test`). A list of available commands will be displayed in [Home Assistant System Logs](https://my.home-assistant.io/redirect/logs).
 
 {% details "Some commonly used commands" %}
```

Comment:
The change needed here, is adding a new-line before and after the liquid syntax (`{% ... %}`, allowing the parser to switch context between liquid and Markdown.